### ISSUE TITLE
text on-top of text in mobile version

### ISSUE THREAD

**Issue by spottedmahn** [terrajobst/themesof.net#22]
![image](https://user-images.githubusercontent.com/14079228/111661523-77acb580-87e5-11eb-9077-d394edb2c419.png)

**Comment by terrajobst** [comment]
Yeah, I noticed that the mobile view is far from idea. That's an area where I need to do some learning. I'm happy for any suggestions or PRs! :-)
